lib/install/node.js

```diff
@@ -41,13 +44,8 @@ var create = exports.create = function (node, template, isNotTop) {
   if (!isNotTop) {
     // isLink is true for the symlink and everything inside it.
     // by contrast, isInLink is true for only the things inside a link
-    if (node.isLink == null && isLink(node.parent)) {
-      node.isLink = true
-      node.isInLink = true
-    } else if (node.isLink == null) {
-      node.isLink = false
-      node.isInLink = false
-    }
+    if (node.isLink == null) node.isLink = isLink(node.parent)
+    if (node.isInLink == null) node.isInLink = isInLink(node.parent)
```

Comment:
The way we were default before meant that nodes couldn't set their isInLink property on their own.